docs/docsite/rst/network/dev_guide/developing_plugins_network.rst

```diff
@@ -0,0 +1,108 @@
+
+.. _developing_modules_network:
+.. _developing_plugins_network:
+
+**************************
+Network connection plugins
+**************************
+
+Each network connection plugin has a set of plugins which provide a specification of the connection for a particular set of devices. Public methods of these plugins may be called on the connection proxy object from the module just as connection methods can.
+
```

Comment:
```suggestion
Because the httpapi architecture uses plugins, modules do not refer directly to their httpapi implemention for requests. Instead, they reference `Connection`, as shown below. `Connection` uses the host's `ansible_network_os` value to determine which plugin implementation to execute for a host.
```

Reply:
I've gone a slight;y different direction, first to hopefully make clearer that this section does is about _module_ code, and not plugins, and second because this document is not specifically about `httpapi`, though that is where we are focusing the most attention right now.

Reply:
I saw the commit you pushed and it was clear. Thank you!